tests/test_cnn/test_wrappers.py

```diff
@@ -105,6 +106,61 @@ def test_conv_transposed_2d():
     wrapper(x_empty)
 
 
+@patch('torch.__version__', '1.1')
+def test_conv_transposed_3d():
+    test_cases = OrderedDict([('in_w', [10, 20]), ('in_h', [10, 20]),
```

Comment:
We could use `@pytest.mark.parametrize()` to manage the test cases and skip the for loop in line 117.